<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsQueryResultWidgetBase</class>
 <widget class="QWidget" name="QgsQueryResultWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
  <item>
   <widget class="QgsMessageBar" name="mMessageBar" native="true"/>
  </item>
   <item>
    <widget class="QgsCodeEditorSQL" name="mCodeEditor" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">    
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mExecuteButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mStopButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="mQueryResultsTableView"/>
   </item>
   <item>
    <widget class="QLabel" name="mStatusLabel">
     <property name="text">
      <string>Status and errors goes here.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCodeEditorSQL</class>
   <extends>QWidget</extends>
   <header>qgscodeeditorsql.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMessageBar</class>
   <extends>QWidget</extends>
   <header>qgsmessagebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
